lib/src/piece/list.dart

```diff
@@ -487,3 +487,10 @@ class ListStyle {
       this.splitListIfBeforeSplits = false,
       this.allowBlockElement = false});
 }
+
```

Comment:
Heh, what's happening with this enum? Probably remove this too ahah

Reply:
Heh, oops! I was seeing how the current formatter behaved with some temp code. Removed.